### ISSUE TITLE
A11y: Number of products on the wish list is not readable by a screen reader

### ISSUE THREAD

**Issue by blacksheep--** [shopware/platform#7107]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The number of products that have already been added to the wish list is not readable by a screen reader. An `aria-label` attribute has been set on the “Whish list” link, which does not contain the information about the number of products. The link already has a descriptive label including the number of products added. The `aria-label` attribute overwrites the labeling of the child elements.

Blind people do not receive any information about how many products have already been added to the wish list.

*_Expected behaviour:_*

Also add the number to the `aria-label`:

![Image](https://github.com/user-attachments/assets/cd58d411-e607-4a82-803a-3e4b836d6503)

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. add a product to the wishlist

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40863**



[created from NEXT-40863, comment 541926]

**Comment by blacksheep--** [comment]
To add: the same applies for the cart and could be solved like this:

![Image](https://github.com/user-attachments/assets/7a6106c2-0c16-4e76-afd7-5d82357b62d5)

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40863, comment 541980]

**Comment by shopwareBot** [comment]
[public] Thank you so much for your very detailed issue report and supporting us in our goal to become more accessible! This issue is already covered by: [https://github.com/shopware/shopware/issues/7084](https://github.com/shopware/shopware/issues/7084)



[created from NEXT-40863, comment 542164]
